infra/src/main/kotlin/fi/fta/geoviite/infra/geography/CoordinateTransformationService.kt

```diff
@@ -0,0 +1,88 @@
+package fi.fta.geoviite.infra.geography
+
+import fi.fta.geoviite.infra.common.Srid
+import fi.fta.geoviite.infra.math.IPoint
+import fi.fta.geoviite.infra.math.Point
+import fi.fta.geoviite.infra.tracklayout.LAYOUT_CRS
+import org.geotools.geometry.jts.JTS
+import org.geotools.referencing.CRS
+import org.opengis.referencing.crs.CoordinateReferenceSystem
+import org.opengis.referencing.operation.MathTransform
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Service
+
+data class Transformation(
```

Comment:
Täällä tiedostossa siis itse pihvi. Myös Transformation itsessään pitää exposata, sillä CsvParsing luo näitä sisäisesti käsin

Reply:
Joo tuo on ihan ok. Helpottaa myös testausta.